tests/TestCase/Identifier/LdapIdentifierTest.php

```diff
@@ -82,6 +82,24 @@ public function testIdentify()
     }
 
     /**
+     * testWrongLdapObject
+     *
+     * @return void
+     * @expectedException \RuntimeException
+     * @expectedExceptionMessage Could not build the LDAP connection object.
+     */
+    public function testWrongLdapObject()
+    {
+        $identifier = new LdapIdentifier([
+            'host' => 'ldap.example.com',
+            'bindDN' => function () {
+                return 'dc=example,dc=com';
+            },
+            'ldapClass' => 1
```

Comment:
OK, but how realistic is this test then? Providing an invalid class name as string sounds to me like the only actual testable thing. Otherwise we would be testing all kind of crazyness as arguments everywere :)